File: C.C++ LIB/write_data.cpp

void write_data(CString name,int* data)
{

	FILE *fout = fopen("..\\_bin\\data.dat","a+"); 
	int i=0;

	fprintf(fout,"%s\t",name);
	while(i<NUM)
	{
		fprintf(fout,"%d:%d\t",i,*(data+i));
		i++;
	}
	fprintf(fout,"\n");
	fclose(fout);
}

File: MY MAT LIB/Leaf/write_data.m
function [] = write_data( filename, ratio)
%%
DIR = 'D:\temp\result.dat';

%%
fid = fopen(DIR,'A');
fprintf(fid,'%s', filename);
fprintf(fid,'\t');
fprintf(fid,'%.5f', ratio);
fprintf(fid,'\n');
fclose(fid);

end

